nix/machines/_common/base.nix

```diff
@@ -36,15 +36,41 @@
     tmux
     usbutils
     unixtools.nettools
-    vim
     wget
+    ((vim_configurable.override {  }).customize{
+      name = "vim";
+      # Install plugins for example for syntax highlighting of nix files
+      vimrcConfig.packages.myplugins = with pkgs.vimPlugins; {
+        start = [ vim-nix vim-lastplace ];
+        opt = [];
+      };
+      vimrcConfig.customRC = ''
+        " Turn on syntax highlighting by default
+        syntax on
+        " Disable mouse
+        set mouse-=a
+      '';
+    })
   ];
 
   # Purge nano from being the default
   environment.variables = {
     EDITOR = "vim";
   };
 
+  # set 24h military time
+  i18n.extraLocaleSettings = {
+    LC_ADDRESS = "en_US.UTF-8";
+    LC_IDENTIFICATION = "en_US.UTF-8";
+    LC_MEASUREMENT = "en_US.UTF-8";
+    LC_MONETARY = "en_US.UTF-8";
+    LC_NAME = "en_US.UTF-8";
+    LC_NUMERIC = "en_US.UTF-8";
+    LC_PAPER = "en_US.UTF-8";
+    LC_TELEPHONE = "en_US.UTF-8";
+    LC_TIME = "C.UTF-8";
```

Comment:
@owendelong can you confirm how to you expected to set military time?

The `LC_TIME` locale setting seems to correctly set the `date` output, but wanted confirmation